sdk/offchain-message/Cargo.toml

```
@@ -0,0 +1,32 @@
+[package]
+name = "solana-offchain-message"
+description = "Solana offchain message signing"
+documentation = "https://docs.rs/solana-offchain-message"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+num_enum = { workspace = true }
+solana-hash = { workspace = true }
+solana-packet = { workspace = true }
+solana-pubkey = { workspace = true, optional = true }
+solana-sanitize = { workspace = true }
+solana-sha256-hasher = { workspace = true }
+solana-signature = { workspace = true }
+solana-signer = { workspace = true }
+
+[dev-dependencies]
+solana-keypair = { workspace = true }
+solana-offchain-message = { path = ".", features = ["dev-context-only-utils"] }
+static_assertions = { workspace = true }
+
+[features]
+dev-context-only-utils = ["verify"]
+verify = ["dep:solana-pubkey", "solana-signature/verify"]
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
```

Since this one has some stuff gated by a feature, how about adding the docs bits?
```suggestion
[package.metadata.docs.rs]
targets = ["x86_64-unknown-linux-gnu"]
all-features = true
rustdoc-args = ["--cfg=docsrs"]
```